src/constants/lists/connections/options.constants.ts

```diff
@@ -21,7 +21,7 @@ export const githubIntegrationAuthMethods: SelectOption[] = [
 	{ label: "PAT + Webhook", value: ConnectionAuthType.Pat },
 ];
 
-const heightDisplayOAuth = featureFlags.heightDisplayDefaultOAuth
+const heightDisplayOAuth = !featureFlags.heightHideDefaultOAuth
 	? [{ label: "OAuth v2 - Default app", value: ConnectionAuthType.OauthDefault }]
 	: [];
```

Comment:
If we can, we prefer to use positive logic in an if block.
Generally, it's easier to read and even understand positive conditions than negative.
So instead of doing `if (!A) then X else Y` we can do `if (A) then Y else X`